ethexe/contracts/test/Base.t.sol

```diff
@@ -121,6 +121,10 @@ contract Base is POCBaseTest {
                             uint256(eraDuration),
                             uint256(electionDuration),
                             uint256(validationDelay),
+                            Gear.AggregatedPublicKey(
+                                0x0000000000000000000000000000000000000000000000000000000000000001,
+                                0x4218F20AE6C646B363DB68605822FB14264CA8D2587FDD6FBC750D587E76A7EE
+                            ),
```

Comment:
this public key on the secp256k1 curve, which is not used now because we don't have the FROSTOffchain library to sign messages from Solidity